gem/lib/metasploit-payloads.rb

```diff
@@ -10,27 +10,29 @@
 module MetasploitPayloads
   EXTENSION_PREFIX      = 'ext_server_'
   METERPRETER_SUBFOLDER = 'meterpreter'
+  USER_DATA_SUBFOLDER   = 'payloads'
 
   #
   # Get the path to an extension based on its name (no prefix).
   #
   def self.meterpreter_ext_path(ext_name, binary_suffix)
```

Comment:
I did a quick search in the metasploit framework and didn't find any references. Since this is a Gem though I didn't want to delete it in case it's used by another project that I'm unaware of. If you think it can safely be removed, I'd be happy to do it.